polaris-react/src/components/UnstyledButton/tests/UnstyledButton.test.tsx

```diff
@@ -212,15 +212,15 @@ describe('<Button />', () => {
     it('passes to `button`', () => {
       const button = mountWithApp(<UnstyledButton disabled />);
       expect(button).toContainReactComponent('button', {
-        disabled: true,
+        'aria-disabled': true,
```

Comment:
Would be good to add a couple of tests here to ensure that the `onClick` and `onKeyPress` props are not getting called if `disabled`. Should be fairly similar to the test you wrote for the `Button.test.tsx` file.